### PR TITLE
1788/fix/validate-identification-number

### PR DESCRIPTION
**Pull request by Noam6421**
Fix to: [1788](https://dev.azure.com/spectrumFactory/CoronaI/_sprints/backlog/CoronaI%20Team/CoronaI/Sprint%2018?workitem=1788)

Now when one of the fields: identificationType, isentificationNumber is filled there will be error message to fill the second field also :)

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonalSchema.ts [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 export const contactQuestioningPersonal = {
     [InteractedContactFields.IDENTIFICATION_NUMBER]: yup
         .string()
         .when(
-            [InteractedContactFields.CONTACT_STATUS, InteractedContactFields.DOES_NEED_ISOLATION],
-            (contactStatus: number, needIsolation: boolean, schema: any, { originalValue }: { originalValue: string }) => {
-                return contactStatus === ContactStatusCodes.COMPLETED || (originalValue === '' && !needIsolation)
+            [InteractedContactFields.CONTACT_STATUS, InteractedContactFields.DOES_NEED_ISOLATION, InteractedContactFields.IDENTIFICATION_TYPE],
+            (contactStatus: number, needIsolation: boolean, identificationType: number, schema: any, { originalValue }: { originalValue: string }) => {
+                return (contactStatus === ContactStatusCodes.COMPLETED ||  !needIsolation) &&  identificationType == null
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonalSchema.ts:15]: why ==?

**Review by y-golde — CHANGES_REQUESTED** [review]
nice work overall

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonalSchema.ts (hunk 1 above)
Comment (Noam6421) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonalSchema.ts:15]: the value of the field changed between null to undefined, ( == null) checks both cases

Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonalSchema.ts:15]: oh, cool

**Review by y-golde — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**